src/sentry/tasks/integrations.py

```diff
@@ -178,3 +182,65 @@ def migrate_repo(repo_id, integration_id, organization_id):
             integration,
             Organization.objects.get(id=organization_id),
         ).call()
+# Particulariliry For VSTS
+# TODO(lb): I want to say do this every 6 hours
+
+
+@instrumented_task(
+    name='sentry.tasks.integrations.kickoff_vsts_subscription_check',
+    queue='integrations',
+    default_retry_delay=60 * 5,  # TODO(lb): not sure what this should be
+    max_retries=5,
+)
+@retry()
+def kickoff_vsts_subscription_check():
+    organization_integrations = OrganizationIntegration.objects.filter(
+        integration__provider='vsts',
+        integration__status=ObjectStatus.VISIBLE,
+        status=ObjectStatus.VISIBLE,
+    ).select_related('integration')
+    six_hours_from_now = time() - mktime(timedelta(hours=6).timetuple())
```

Comment:
Definitely nitpicky, but two things.

First, since you're dealing with whole hours/minutes/seconds, you can just do,

```python
time() + timedelta(hours=6).seconds
```

I believe. Seems a little more readable to me. (sidenote: surely there has to be a python library out there that makes date/time math more readable...)

Second, `from_now` implies the future to me, whereas `ago` would imply the past (like you're generating here). I think calling this `six_hours_ago` makes more sense.